lms/djangoapps/onboarding/choices.py

```diff
@@ -0,0 +1,50 @@
+FUNCTIONS_LABELS = (
+    ('strategy_planning', 'Strategy and planning'),
+    ('leadership_governance', 'Leadership and governance'),
+    ('program_design', 'Program design and development'),
+    ('measurement_eval', 'Measurement evaluation and learning'),
+    ('stakeholder_engagement', 'External relations and partnerships'),
+    ('human_resource', 'Human resource management'),
+    ('financial_management', 'Financial management'),
+    ('fundraising', 'Fundraising and resource mobilization'),
+    ('marketing_communication', 'Marketing communications and PR'),
+    ('system_tools', 'Systems tools and processes'),
+)
+
+INTERESTS_LABELS = (
+    ('strategy_planning', 'Strategy and planning'),
+    ('leadership_governance', 'Leadership and governance'),
+    ('program_design', 'Program design and development'),
+    ('measurement_eval', 'Measurement, evaluation, and learning'),
+    ('stakeholder_engagement', 'External relations and partnerships'),
+    ('human_resource', 'Human resource management'),
+    ('financial_management', 'Financial management'),
+    ('fundraising', 'Fundraising and resource mobilization'),
+    ('marketing_communication', 'Marketing, communications, and PR'),
+    ('system_tools', 'Systems, tools, and processes'),
+)
+
+INTERESTED_LEARNERS_LABELS = (
+    ('same_region', 'Learners from my region or country'),
+    ('similar_oe_interest', 'Learners interested in same areas of organization effectiveness'),
+    ('similar_org', 'Learners working for similar organizations'),
+    ('diff_who_are_different', 'Learners who are different from me')
+)
+
+GOALS_LABELS = (
+    ('contribute_to_org', 'Help improve my organization'),
+    ('gain_new_skill', 'Develop new skills'),
+    ('improve_job_prospect', 'Get a job'),
+    ('relation_with_other', 'Build relationships with other nonprofit leaders')
+)
+
+HEAR_ABOUT_PHILANTHROPY_LABELS = (
```

Comment:
Please rename all the constants => Replace `_LABELS` with `_CHOICES`
As in previous implementation all constants were used for Field Labels but now these are used for choices ..

Reply:
Secondly Please make sure keys in all constants stays same with field names ignoring the prefixes. So that when we ll be writing migrations, It would be easy to map old fields data with new fields.  



We can discuss this on call for more clarity. Thanks.

Reply:
I have removed the suffix 'LABELS' all together instead of replacing it with 'CHOICES', because the choices are already placed in a file 'choices.py' and appending 'CHOICES' at the end would be redundant.